Update Move-CsUser.md (#11272)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Move-CsUser.md

> - Moving users from On-Premises to Teams requires TLS 1.2. TLS 1.0 and TLS 1.1 have been deprecated. Please visit [Disabling TLS 1.0 and 1.1 for Microsoft 365](/microsoft-365/compliance/tls-1.0-and-1.1-deprecation-for-office-365?view=o365-worldwide) and [Preparing for TLS 1.2 in Office 365 and Office 365 GCC](/microsoft-365/compliance/prepare-tls-1.2-in-office-365?view=o365-worldwide) for details. 
> - To use Multi-Factor Authentication (MFA) with Move-CsUser requires either Skype for Business Server 2015 CU12 or any version of Skype for Business Server 2019. When using MFA do not specify the -Credential paremeter. If you are using an earlier version of Skype for Business Server, you should either disable MFA and use the credential parameter, or obtain a newer version of the administrative tools for Skype for Business Server that supports MFA.

> [!NOTE]

> >
> > - From November 10, 2023 onwards, moving users from Teams to On-Premises will discontinue migrating their contacts. This is mainly due to Microsoft's continuous efforts to tighten the security and protect customer's data. After carefully analyzing the usage patterns and performing risk assessment with the legacy infrastructure, Microsoft decided to deprecate this feature. 

Reviewer: ```suggestion
> As of November 10, 2023, moving users from Teams to an on-premises deployment will no longer migrate their contacts. This is mainly due to our continuous efforts to tighten security and protect customers' data. After carefully analyzing the usage patterns and performing risk assessments with the legacy infrastructure, we decided to deprecate this feature. 
```
